R/opart_poission.R

```diff
@@ -0,0 +1,63 @@
+#' compute the optimal changepoint model for a vector of real-valued data
```

Comment:
Hi @tdhock ,

I have started creating function for poisson loss function. I am going with creating a separate .R and .C files as well as interface for poisson loss so that 2 different functions are available under package opart (opart_gaussian and opart_poisson). Does this approach seems fine or should I use the same interface with a parameter indicating which loss function to use ?